docs/fuel-101/fuel-virtual-machine.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Fuel Virtual Machine
+category: Fuel 101
+---
+
+# Fuel Virtual Machine (FVM)
+
+> The Fuel Virtual Machine is an operating system designed for unparalleled speed in generalized computation on the execution layer.
```

Comment:
I wonder if we want to change 'generalized computation' to blockchain computation or something similar?

Nick [often talks about](https://youtu.be/_VAzYP02-xA?si=f7fb1eMyDD5e36Uw&t=204) how other architectures are not fit for blockchain because they are for general computing (e.g. video games, web browsing, etc.) and the Fuel VM is purpose built for blockchain. What do you thinl?

Reply:
Great point let's change it